scripts/qa_crowbarsetup.sh

```diff
@@ -2489,6 +2522,12 @@ EOF
         fi
     fi
 
+    # prepare docker image at sle12 compute node
+    if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
+        local sle12node=$(knife search node "target_platform:suse-12.0" -a name | grep ^name: | cut -d : -f 2 | tail -n 1 | sed 's/\s//g')
```

Comment:
Duplicated code.
For such tasks I think it is worth addin a function. Then the code here gets more readable and the magic can be done in a separate function. Plus, it can be reused by other people and it can be tested individually.